server/apps/users/views.py

```diff
@@ -106,21 +84,36 @@ class LogoutView(APIView):
     @staticmethod
     def post(request):
         response = Response()
+        blacklist_jwt_token(request.COOKIES.get('jwt'))
         response.delete_cookie('jwt')
+        response.delete_cookie('csrftoken')
         response.data = {
             'message': LOGOUT_SUCCESSFUL
         }
         return response
 
 
-class TodoListView(APIView):
+class TodoView(generics.CreateAPIView, generics.RetrieveUpdateDestroyAPIView):
+    # Create to'do
     @staticmethod
```

Comment:
Why are these functions defined as staticmethods?

Reply:
Fixed it.